cpp/src/gandiva/precompiled/extended_math_ops.cc

```diff
@@ -122,8 +138,8 @@ POWER(float64, float64, float64)
         scale_multiplier);                                                  \
   }
 
-ROUND_DECIMAL(float32)
-ROUND_DECIMAL(float64)
+ROUND_DECIMAL_TWO_ARG(float32)
```

Comment:
can we rename this to say something like 'round_decimal_to_scale'?

Reply:
Addressed the comments.